spec/features/cert_displays_spec.rb

```diff
@@ -32,7 +32,7 @@
       expect(page).to have_link(person.fullname)
       expect(page).to have_link(cert.course.name)
       click_on person.fullname
-      expect(page).to have_content('CJ')
+      expect(page).to have_content('Cj')
```

Comment:
??

Reply:
@kgf  in order to properly format fullnames, I am using the capitalize method which uppercases the first letter and lower for the rest.

Reply:
I don't know that changing the capitalization from what was entered is a good idea. What about names like MacDonald ?